firebase-project/src/components/navbar.vue

```diff
@@ -34,11 +35,15 @@ export default {
   setup() {
     const router = useRouter();
 
-    const isLoginPage = computed(() => {
+    const isLoginRoute = computed(() => {
```

Comment:
@mukeshsin Create a composable api for this one, from that you can directly computed export  `router.currentRoute.value.path` then you can use anywhere it in you app, without doing all 
```
    const isRegisterRoute = computed(() => {
      return router.currentRoute.value.path === "/register";
    });
    ```